### DISCUSSION TITLE
Editor is extremely laggy

### DISCUSSION THREAD

**Discussion by YoiSakura** [ppy/osu#23293] [Q&A] [answered]
<img width="90" alt="Screenshot 2023-04-23 at 8 38 37 PM" src="https://user-images.githubusercontent.com/110989599/233840296-37fcbdcc-8779-4a41-88b1-c95abef828a2.png">
I have been charting an osu!mania map, just to start off.

I have noticed that objects are really taking up a lot of performance. Basically, the more objects you have (like timing and notes and stuff) are just starting to lag my computer a lot more. To be specific, I'm using a 16 core Apple Silicon GPU. 

I've also experienced a couple of sudden lagspikes while editing

The FPS is just worse than the one when you play the map. I don't really know anything about lag causes and stuff so pardon me.

**Comment by peppy** [comment, marked as answer]
141fps is higher than your refresh rate. This is ample for the editor.
